### ISSUE TITLE
SS AP Launcher inconsistent

### ISSUE THREAD

**Issue by vincentzhang96** [ProjectDuckDN/IssueTracker#74]
Shooting Star has a bug where when right clicking after any skill, sometimes it will cast the old Lock and Load animation and put AP Launcher on CD without doing the dmg of AP Launcher (from what I can tell). I've noticed it most consistently when trying to Rocket Crash after tumbling.

https://discord.com/channels/544827049752264704/803135119665266718/974081398908534835

**Comment by wellme** [comment]
"Update: Lock and Load activates through the old "right click after any skill" activation method when AP Launcher is off cooldown.

Only occurs after 2 skills: Roto-Whirl (tumble, Rocket Crash must be on CD/not taken), and Make it Rain EXi. Does not occur after MiR, only after EXi.

Skill casts exactly the same as Lock and Load, and deals the total damage of 1 hit of AP Launcher" - Salvani#3389

**Comment by vincentzhang96** [comment]
Probably no longer relevant with upcoming SS/GM changes (whenever Austin's back)
